### ISSUE TITLE
Shapes Scores

### ISSUE THREAD

**Issue by jforsyth** [mitchellss/real_time_pose#8]
Score appears to constantly increment between runs. Should reset?

**Comment by mitchellss** [comment]
@jforsyth Yes it should reset. Would you rather the score reset every round or just not be included at all?

**Comment by jforsyth** [comment]
Let's reset after each trial. This may be my only time to get a pull request on this repo.


_____________________________
Jason Forsyth, PhD
Associate Professor of Engineering
Department of Engineering
James Madison University
www.jasonforsyth.net<http://www.jasonforsyth.net/>


________________________________
